deploy/roles/application/tasks/cron.yml

```diff
@@ -5,3 +5,13 @@
     name: "import latest aria-at tests"
     minute: "15"
     job: "curl -X POST https://{{fqdn}}/api/test/import"
+  when: deployment_mode != 'development' 
+
+# The job is written to change the permissions of the /vagrant/server
+# directory because the POST request does not have permission to it
```

Comment:
I'm confused about this. Since the `curl` utility is issuing an HTTP request to the web server, it doesn't seem like it needs access to any particular directories on the file system.

Reply:
@jugglinmike nice catch! I should have worded this differently. The process that is invoked through the POST request accesses the /vagrant/server folder.

Reply:
I see that you've removed the mode changes from the command, but the comment still describes the earlier version.